server/mustache-templates/newClient.mst

```diff
@@ -69,13 +69,49 @@
                         <small id="heartbeatPhoneNumbersHelp" class="form-text text-muted">Each phone number has +1 in front and separated by commas with no dashes or spaces, please (eg. +14445556789,+12223334444,+11231231234)</small>
                     </div>
                 </div>
-                <div class="form-group row justify-content-start row-no-gutters">
-                    <label for="incidentCategories" class="col-sm-3 col-form-label">Incident Categories:</label>
-                    <div class="col-sm-5">
-                        <input type="text" class="form-control" name="incidentCategories" value="{{incidentCategories}}" required>
-                        <small id="incidentCategoriesHelp" class="form-text text-muted">Each incident category is separated by commas with no quotes, please (eg. No One Inside,Person responded,Overdose,None of the above)</small>
-                    </div>
-                </div>
+<div class="form-group row justify-content-start row-no-gutters">
+    <label for="language" class="col-sm-3 col-form-label">Language:</label>
+    <div class="col-sm-5">
+        <select class="form-control" name="language" id="languageSelector" required>
+            <option value='en'>English</option>
+            <option value='es_us'>Spanish</option>
+            <option value='en'>English with Manual Input for Incident Categories</option>
+        </select>
```

Comment:
Do we offer all clients the option to have their own incident categories? I know there was the one example of a client that didn't want to use the term "overdose" but I'm curious if that was a one off and exception we made for only that client, or if this is something we can offer to every client upon request?

Reply:
For English language selection, there is an option to manually fill in these incident categories if the automated fields don't cover the specific case

Reply:
Yes I know that, I guess my question was more about if choosing different incident categories was a common occurrence for clients (or if we prefer clients to use the incident categories we use by default). This question doesn't affect your code, I was just personally curious, so I can ask someone else about this, thanks!